core/src/main/java/org/camunda/bpm/engine/test/assertions/bpmn/ProcessInstanceAssert.java

```diff
@@ -373,6 +373,24 @@ public ProcessInstanceAssert hasProcessDefinitionKey(String processDefinitionKey
     return this;
   }
 
+  /**
+   * Verifies the expectation that the {@link ProcessInstance} has the
+   * given businessKey.
+   *
+   * @param businessKey the expected key
+   * @return  this {@link ProcessInstanceAssert}
+   */
+  public ProcessInstanceAssert hasBusinessKey(String businessKey) {
+    isNotNull();
+    final String message = "Expecting %s to have business key '%s', " +
+      "but instead found it to have business key key '%s'.";
```

Comment:
There is one extra "key" here, right?
```suggestion
      "but instead found it to have business key '%s'.";
```

Reply:
Thanks, fixed it